### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by martinconstante**
Modify to use params to store vendor system data.

### CODE REVIEW

File: examples/apiary_scenarios/asset_accept_workflow.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+      };
+      const approveRequests = async () => {
+        const responseApprove = await fulfillment.approveRequest(element.id, body);
+        return responseApprove;
+      };
+      approveRequests()
+        .then((responseApprove) => console.log(responseApprove))
+        .catch((e) => console.log(e));
+    } else {
+      throw Error('This Tenant not exist in vendor');
```
Comment (ffaraoneim) [inline examples/apiary_scenarios/asset_accept_workflow.js:63]: throw new Error (it is a class it must be instantiated with the new operator).

File: examples/apiary_scenarios/asset_request_workflow.js [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+ * Create tenant into Vendor System
+ *
+ * @param   {object}  element  The request of Cloudblue Connect.
+ *
+ */
+function createTenant(element) {
+  let mpn;
+  let quantity;
+  if (element.asset.items && (element.asset.items.length === 1)) {
+    Object.values(element.asset.items).forEach((item) => {
```
Comment (ffaraoneim) [inline examples/apiary_scenarios/asset_request_workflow.js:42]: Please check 'element.asset.items' it should be an array. If true please use element.asset.items.forEach !

File: examples/apiary_scenarios/asset_request_workflow.js [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+      mpn = item.mpn;
+      quantity = item.quantity;
+    });
+  } else {
+    throw Error('Malformed request, bad quantity of item');
```
Comment (ffaraoneim) [inline examples/apiary_scenarios/asset_request_workflow.js:47]: Please use 'new' to create the 'Error' class.

File: examples/apiary_scenarios/asset_request_workflow.js [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify(payload),
+    cache: 'no-cache',
+  })
+    .then((response) => response.json())
+    .then((data) => {
+      element.asset.params.forEach((param) => {
+        if (param.id === 'tenantId') {
+          fulfillment.updateRequestParameters(element.id, [{ id: 'tenantId', value: data.tenantId }], 'Id of vendor system tenant');
```
Comment (ffaraoneim) [inline examples/apiary_scenarios/asset_request_workflow.js:78]: The updateRequestParameters is async. You must await it.

File: examples/apiary_scenarios/asset_accept_workflow.js (hunk 1 above)
Comment (martinconstante) [reply examples/apiary_scenarios/asset_accept_workflow.js]: Fixed.

File: examples/apiary_scenarios/asset_request_workflow.js (hunk 2 above)
Comment (martinconstante) [reply examples/apiary_scenarios/asset_request_workflow.js]: Fixed

File: examples/apiary_scenarios/asset_request_workflow.js (hunk 3 above)
Comment (martinconstante) [reply examples/apiary_scenarios/asset_request_workflow.js]: Fixed

File: examples/apiary_scenarios/asset_request_workflow.js (hunk 4 above)
Comment (martinconstante) [reply examples/apiary_scenarios/asset_request_workflow.js]: Fixed

**Pull request merged**